OSPSuite.FuncParser.sln

```diff
@@ -29,20 +29,30 @@ Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
 		Debug|x64 = Debug|x64
+		Purify_Debug|Any CPU = Purify_Debug|Any CPU
+		Purify_Debug|x64 = Purify_Debug|x64
```

Comment:
Just added a new build configuration for Purify.
Different profilers require different settings (e.g. the Visual Leak Detector requires `Whole Program Optimization = YES`, but Purify/Quantify Require `Whole Program Optimization = NO`, so I decied to create a new one)
It will not be built on AppVeyor - just for local profiling